src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -31,6 +31,11 @@
     @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME, shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME, doc = "A VCF/BCF file containing variants", common = false, optional = false)
     public File drivingVariantFile;
 
+    /**
+     * This number dictates the number of lookeahead bases of the cache for the primary and the auxiliary features.
```

Comment:
"This number controls the size of the cache for our primary and auxiliary FeatureInputs (specifically, the number of additional bases worth of overlapping records to cache when querying feature sources)."